app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt

```diff
@@ -94,4 +99,30 @@ class IncidentUnitTest {
         assertTrue(containsIncidentWithCaseId400132)
     }
 
+    @Test
```

Comment:
Consolidated unit tests into one file; easier to keep track of, reducing technical debt

Reply:
Did you delete the old ones?  I don't see anything removed.

Also, do you have a source for how this reduces technical debt?  We typically prefer a large number of small classes over a small number of large classes... so I would go in the other direction.